include/Macro.h

```diff
@@ -1012,6 +1012,109 @@
 #  undef STORE_PAR_ACC
 #endif
 
+// wrappers for fftw2 and fftw3 interfaces
+#ifdef SUPPORT_FFTW
 
+// wrappers for fftw3 single and double precision routines
+#ifdef SUPPORT_FFTW3
+#ifdef FLOAT8
+#define gamer_float_fftw3_malloc              fftw_malloc
+#define gamer_float_fftw3_free                fftw_free
+#define gamer_float_fftw3_plan                fftw_plan
+#define gamer_float_fftw3_destroy_plan        fftw_destroy_plan
+#define gamer_float_fftw3_complex             fftw_complex
+#define gamer_float_fftw3_execute_dft_r2c     fftw_execute_dft_r2c
+#define gamer_float_fftw3_execute_dft_c2r     fftw_execute_dft_c2r
+#define gamer_float_fftw3_execute_dft_c2c     fftw_execute_dft
+#define gamer_float_fftw3_plan_dft_r2c_3d     fftw_plan_dft_r2c_3d
+#define gamer_float_fftw3_plan_dft_c2r_3d     fftw_plan_dft_c2r_3d
+#define gamer_float_fftw3_plan_dft_c2c_3d     fftw_plan_dft_3d
+#define gamer_float_fftw3_cleanup             fftw_cleanup
```

Comment:
Yes, added these definitions with the spectral kinetic FDM solver in mind. Of course, one could have another conditional check for it. But I do not think they harm since they are just aliases and do not slow down the compilation or increase the binary size.